wbooks/App.js

```diff
@@ -1,28 +1,43 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow
- */
-
-import React from 'react';
-import {
-  SafeAreaView,
-  StyleSheet,
-  View,
-  Text,
-  StatusBar,
-} from 'react-native';
+import React, { Component } from 'react';
+import { SafeAreaView, StyleSheet, View, Text, StatusBar, Image } from 'react-native';
 import { createAppContainer } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
+import { createBottomTabNavigator } from 'react-navigation-tabs';
 
+import BookListActiveIcon from './src/assets/ic_library_active.png';
+import BookListIcon from './src/assets/ic_library.png';
+import SettingsActiveIcon from './src/assets/ic_settings_active.png';
+import SettingsIcon from './src/assets/ic_settings.png';
 import BookList from './src/app/screens/BookList';
 import BookDetail from './src/app/screens/BookDetail';
+import SettingsScreen from './src/app/screens/Settings';
 import CustomHeader from './src/app/components/Header';
+import TabBarIcon from './src/app/components/TabBar/components/TabBarIcon';
 
 const MainNavigator = createStackNavigator({
-  Home: { screen: BookList },
+  Home: { screen: createBottomTabNavigator(
+    {
+      Home: { 
+        screen: BookList,
+        navigationOptions: {
+          tabBarLabel: null,
+          tabBarIcon: ({focused}) => <TabBarIcon activeIcon={BookListActiveIcon} inactiveIcon={BookListIcon} label={'Library'} focused={focused}/>
+        } 
+      },
+      Settings: {
+        screen: SettingsScreen,
+        navigationOptions: {
+          tabBarLabel: null,
+          tabBarIcon: ({focused}) => <TabBarIcon activeIcon={SettingsActiveIcon} inactiveIcon={SettingsIcon} label={'Settings'} focused={focused}/>
+        }
+      },
+    },
+    {
+      tabBarOptions: {
+        showLabel: false
+      },
+    }
```

Comment:
extract all navigationOptions and tabBarOptions in a file located in the folder `config`. The file must be called `navigationOptions.js`

Reply:
done